woocommerce/woocommerce-android
# Use padding instead of margin for the clickable "learn more" section

A small visual change for the "learn more" section of the manage card reader screen. Now ripple effect covers the whole width.

Before:
<img src="https://user-images.githubusercontent.com/4923871/124448866-8f577c00-dd8b-11eb-9699-8f29bb3ee4d1.png" height="900" />

After:
<img src="https://user-images.githubusercontent.com/4923871/124448909-98484d80-dd8b-11eb-8912-ed94b3de5e88.png" height="900" />
<img src="https://user-images.githubusercontent.com/4923871/124448903-97172080-dd8b-11eb-82bc-bc69ff773666.png" height="900" />

### How to test
* Open manage card reader screen
* Press and hold the "learn more" section
* Notice ripple effect coverage
